doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -496,6 +496,7 @@ Zigbee samples
 Wi-Fi samples
 -------------
 
+* Added :ref:`wifi_scan_sample` that demonstrates how to scan for the access points without using the wpa_supplicant.
```

Comment:
replace "that demonstrates how to scan for the access points without using the wpa_supplicant" with "extended to support new scan APIs"